packages/page-claims/src/index.tsx

```diff
@@ -74,8 +74,9 @@ const ClaimsApp = (props: Props): React.ReactElement => {
   const [signature, setSignature] = useState<EcdsaSignature | null>(null);
   const [step, setStep] = useState<Step>(Step.Account);
   const [accountId, setAccountId] = useState<string | null>(null);
-  const { api, systemChain } = useApi();
+  const { api, isApiReady, systemChain } = useApi();
   const { t } = useTranslation();
+  const [isPreclaimed, setIsPreclaimed] = useState(false);
```

Comment:
```suggestion
  const isPreclaimed = useCall<boolean>(api.query.claims.preclaims, [accountId], {
    transform: (option: Option<EthereumAddress>) => option.isSome
  });
```

1. isApiReady is (mostly) never applicable since components aren't rendered before it is (some exceptions on outer layers)
2. prefer never using the APIs like below, we have `useCall` (once again, there are some exceptions, especially in custom hooks and with stuff like `queryMulti` which is not great)